misc/python/materialize/config.py

```diff
@@ -0,0 +1,147 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
```

Comment:
This module seems a little too generically named—can we call it `build_config` or somesuch?

Reply:
Makes sense, updated!